addons/website_event_track/views/event_track_templates_page.xml

```diff
@@ -37,8 +37,13 @@
                 Event <span t-esc="track.event_id.name" class="font-weight-bold"/>
                 <span t-if="track.event_id.start_today">
                     starts in
-                    <span t-esc="track.event_id.start_remaining"
-                        t-options="{'widget': 'duration', 'digital': False, 'unit': 'minute', 'round': 'minute'}"/>
+                    <t t-if="track.event_id.start_remaining >= 1">
```

Comment:
shouldn't you do `&gt;=` instead of `>=` ? (to have valid XML, same remark for other part of the code)